Source/carthage/Build.swift

```diff
@@ -45,6 +59,7 @@ public struct BuildCommand: CommandProtocol {
 				<*> mode <| Option(key: "verbose", defaultValue: false, usage: "print xcodebuild output inline")
 				<*> mode <| Option(key: "project-directory", defaultValue: FileManager.default.currentDirectoryPath, usage: "the directory containing the Carthage project")
 				<*> mode <| Option(key: "log-path", defaultValue: nil, usage: "path to the xcode build output. A temporary file is used by default")
+				<*> mode <| Option(key: "archive", defaultValue: false, usage: "don't skip building the Carthage project (in addition to its dependencies) and archive")
```

Comment:
I think this description could use a little work.

Maybe:

```
"archive built frameworks from the current project (implies --no-skip-current)"
```

?

Reply:
This is a good one! Updating it as we speak.